api/admin/service.md

```diff
@@ -1,53 +1,40 @@
----
-tags: [AvalancheGo APIs]
-description: This page is an overview of the Admin API associated with AvalancheGo.
-sidebar_label: Admin API
-pagination_label: Admin API
----
+The Admin API can be used for measuring node health and debugging.
 
-# Admin API
+<Callout title="Note">
+The Admin API is disabled by default for security reasons. To run a node with the Admin API enabled, use [`config flag --api-admin-enabled=true`](/nodes/configure/configs-flags#--api-admin-enabled-boolean).
 
-This API can be used for measuring node health and debugging.
-
-:::info
-The Admin API is disabled by default for security reasons. To run a node with the Admin API
-enabled, use [config flag `--api-admin-enabled=true`](/nodes/configure/avalanchego-config-flags.md#--api-admin-enabled-boolean).
-
-This API set is for a specific node, it is unavailable on the [public server](/tooling/rpc-providers.md).
-
-:::
+This API set is for a specific node, it is unavailable on the [public server](/tooling/rpc-providers).
+</Callout>
 
 ## Format
 
-This API uses the `json 2.0` RPC format. For details, see [here](/reference/standards/guides/issuing-api-calls.md).
+This API uses the `json 2.0` RPC format. For details, see [here](/api-reference/standards/guides/issuing-api-calls).
 
 ## Endpoint
 
-```text
+```sh
```

Comment:
This isn't `sh` - is there a better alternative than `sh`? perhaps nothing?

Reply:
fixed